pkg/straceback/syscall_table.go

```diff
@@ -2,340 +2,340 @@
+	syscallNames[249] = "request_key"
+	syscallNames[250] = "keyctl"
+	syscallNames[251] = "ioprio_set"
+	syscallNames[252] = "ioprio_get"
+	syscallNames[253] = "inotify_init"
+	syscallNames[254] = "inotify_add_watch"
+	syscallNames[255] = "inotify_rm_watch"
+	syscallNames[256] = "migrate_pages"
+	syscallNames[257] = "openat"
+	syscallNames[258] = "mkdirat"
+	syscallNames[259] = "mknodat"
+	syscallNames[260] = "fchownat"
+	syscallNames[261] = "futimesat"
+	syscallNames[262] = "newfstatat"
+	syscallNames[263] = "unlinkat"
+	syscallNames[264] = "renameat"
+	syscallNames[265] = "linkat"
+	syscallNames[266] = "symlinkat"
+	syscallNames[267] = "readlinkat"
+	syscallNames[268] = "fchmodat"
+	syscallNames[269] = "faccessat"
+	syscallNames[270] = "pselect6"
+	syscallNames[271] = "ppoll"
+	syscallNames[272] = "unshare"
+	syscallNames[273] = "set_robust_list"
+	syscallNames[274] = "get_robust_list"
+	syscallNames[275] = "splice"
+	syscallNames[276] = "tee"
+	syscallNames[277] = "sync_file_range"
+	syscallNames[278] = "vmsplice"
+	syscallNames[279] = "move_pages"
+	syscallNames[280] = "utimensat"
+	syscallNames[281] = "epoll_pwait"
+	syscallNames[282] = "signalfd"
+	syscallNames[283] = "timerfd_create"
+	syscallNames[284] = "eventfd"
+	syscallNames[285] = "fallocate"
+	syscallNames[286] = "timerfd_settime"
+	syscallNames[287] = "timerfd_gettime"
+	syscallNames[288] = "accept4"
+	syscallNames[289] = "signalfd4"
+	syscallNames[290] = "eventfd2"
+	syscallNames[291] = "epoll_create1"
+	syscallNames[292] = "dup3"
+	syscallNames[293] = "pipe2"
+	syscallNames[294] = "inotify_init1"
+	syscallNames[295] = "preadv"
+	syscallNames[296] = "pwritev"
+	syscallNames[297] = "rt_tgsigqueueinfo"
+	syscallNames[298] = "perf_event_open"
+	syscallNames[299] = "recvmmsg"
+	syscallNames[300] = "fanotify_init"
+	syscallNames[301] = "fanotify_mark"
+	syscallNames[302] = "prlimit64"
+	syscallNames[303] = "name_to_handle_at"
+	syscallNames[304] = "open_by_handle_at"
+	syscallNames[305] = "clock_adjtime"
+	syscallNames[306] = "syncfs"
+	syscallNames[307] = "sendmmsg"
+	syscallNames[308] = "setns"
+	syscallNames[309] = "getcpu"
+	syscallNames[310] = "process_vm_readv"
+	syscallNames[311] = "process_vm_writev"
+	syscallNames[312] = "kcmp"
+	syscallNames[313] = "finit_module"
+	syscallNames[314] = "sched_setattr"
+	syscallNames[315] = "sched_getattr"
+	syscallNames[316] = "renameat2"
+	syscallNames[317] = "seccomp"
+	syscallNames[318] = "getrandom"
+	syscallNames[319] = "memfd_create"
+	syscallNames[320] = "kexec_file_load"
+	syscallNames[321] = "bpf"
+	syscallNames[322] = "execveat"
+	syscallNames[323] = "userfaultfd"
+	syscallNames[324] = "membarrier"
+	syscallNames[325] = "mlock2"
+	syscallNames[326] = "copy_file_range"
+	syscallNames[327] = "preadv2"
+	syscallNames[328] = "pwritev2"
+	syscallNames[329] = "pkey_mprotect"
+	syscallNames[330] = "pkey_alloc"
+	syscallNames[331] = "pkey_free"
+	syscallNames[332] = "statx"
+	syscallNames[333] = "io_pgetevents"
+	syscallNames[334] = "rseq"
```

Comment:
On my system with `kernel-headers-5.5.5-200.fc31.x86_64`, the list in `/usr/include/asm/unistd_64.h` continues until:
```
#define __NR_clone3 435
```
Maybe you can run `scripts/get_syscall_table.sh` at the same time.

Reply:
Let's do it in the follow up PR too.